Lab1.ipynb

```diff
@@ -241,15 +249,256 @@
+      "text/plain": [
+       "<Figure size 1080x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "n = 10\n",
+    "p = .9\n",
+    "# defining list of r values \n",
+    "r_values = list(range(n + 1)) \n",
+    "# list of pmf values \n",
+    "dist = [binom.pmf(r, n, p) for r in r_values ] \n",
+    "# plotting the graph  \n",
+    "plt.bar(r_values, dist) \n",
+    "plt.xlabel('X', fontsize = 27) \n",
+    "plt.ylabel('P(x)', fontsize = 27)\n",
+    "plt.title('Bionomial Distribution w/ n = 10, p = .9', fontsize = 30)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Seen in the code above with a binomial distrbution, it's almost impossible to succeed less than five times in a 10 trial grouping. "
```

Comment:
Part 2 Problem 1B:
- almost impossible *due to the p=0.9 parameter right? emphasize the impact p has on the graph